src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -254,7 +274,25 @@ private Result executeCmd(String cmd) throws IOException, InterruptedException {
                     .stderr(stderr)
                     .cmdAsSingleString(cmd)
                     .start().join() == 0;
-            String stdoutStr = stdout.toString();
+
+            // capture the stdout so it can be parsed later on
+            String stdoutStr = null;
+            try {
+                Computer computer = Computer.currentComputer();
+                if (computer != null) {
+                    Charset charset = computer.getDefaultCharset();
+                    if (charset != null) {
+                        baos.flush();
```

Comment:
is there anything that can be done here out of the launcher?
in this case is not particularly important, if the output can't be read in some special cases, it just triggers an extra command